src/tests/userController.spec.ts

```diff
@@ -395,12 +420,54 @@ describe('User controller tests', () => {
 		});
 
 		sinon.stub(UserService, 'getById').resolves(mockUsers[0]);
+		sinon.stub(UserService, 'getTutorByUserId').resolves(undefined);
+		sinon.stub(UserService, 'saveTutor').resolves(mockTutorInfo);
 		sinon.stub(UserService, 'save').resolves();
 		sinon.stub(validator, 'validate').resolves([]);
 		await UserController.editTutorRole(req, res);
 		expect(res.statusCode).to.equal(200);
 	});
+	it('should fali to change tutor role to TUTOR when a tutorinfo does not exists and return 400 because it fails to save', async () => {
```

Comment:
```suggestion
	it('should fail to change tutor role to TUTOR when a tutorinfo does not exists and return 400 because it fails to save', async () => {
```